src/lib/sidebar/ListMode.svelte

```diff
@@ -90,30 +90,30 @@
   }
 </script>
 
-<SecondaryButton on:click={newBlankScheme}>
-  Add new blank scheme
-</SecondaryButton>
-<FileInput label="Add scheme from file" onLoad={mergeSchemesFromFile} />
+<div style="display: flex; justify-content: space-between;">
```

Comment:
Styling here is a little funky; I'm going to make a big pass through everything after some of these big PRs and really make the margins nice